build/local/README.md

```diff
@@ -1,3 +1,13 @@
+## Prerequisites 
+Prior to following the instructions here, make sure of the following steps to ensure a smooth process - 
+ - You should have `go` installed on your machine. You can find the installation instructions for your OS [here](https://go.dev/doc/install). 
+    - Verify the installation by running `go version`.
+    - Add the path to go's installation `bin` folder to your PATH variable. Instructions to do this are also mentioned on the go installation instructions page.  
+ - Make sure that you have [docker](https://docs.docker.com/engine/install/) installed on your machine.
+    - Also verify that you are able to run docker commands without the need to gain root permissions using sudo. You can follow the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to achieve this.
+    - To verify that the post-installation steps have been completed successfully and that you can run docker without the need for using `sudo`, try running `docker run hello-world`. This command should run successfully now. 
```

Comment:
don't need this verify step

Reply:
I can remove the other verification steps, but I do think that we should keep the post-installation steps sub-point as-is.  

That step is optional for docker installation, but necessary to run these samples.

Reply:
Sorry, I wasn't clear... I think the post-install step is good to keep, but the verify step we don't need